cartographer/mapping/map_builder.cc

```diff
@@ -168,7 +168,10 @@ std::string MapBuilder::SubmapToProto(
   return "";
 }
 
-void MapBuilder::SerializeState(io::ProtoStreamWriterInterface* const writer) {
+void MapBuilder::SerializeState(io::ProtoStreamWriterInterface* const writer,
+                                const std::set<int> trajectory_ids) {
```

Comment:
Should this by any chance be a `const std::set<int>&` ?

Reply:
Yes. The params will change anyway as I used a default parameter value in the `map_builder_interface.h` in this prototype, which is not conform to the Google style guide.